tests/test.py

```diff
@@ -4,23 +4,29 @@
 
 parser = WiktionaryParser()
 
-class TestParser(unittest.TestCase): 
+class TestParser(unittest.TestCase):
     def test_multiple_languages(self):
         sample_output = {}
         with open('tests/testOutput.json', 'r') as f:
             sample_output = json.load(f)
         words_to_test = {
-            'English': ['grapple', 'test', 'patronise', 'abiologically', 'alexin', 'song', 'house'],
-            'Latin': ['video'],
-            'Norwegian Bokmål': ['seg', 'aldersblandet', 'by', 'for', 'admiral', 'heis', 'konkurs', 'pantergaupe', 'maldivisk'],
-            'Swedish': ['house'],
-            'Ancient Greek': ['ἀγγελία']
+            'English': {'grapple': 50080840, 'test': 50342756, 'patronise': 49023308, 'abiologically': 43781266, 'alexin': 50152026, 'song': 50235564, 'house': 50356446},
+            'Latin': {'video': 50291344},
+            'Norwegian Bokmål': {'seg': 50359832, 'aldersblandet': 38616917, 'by': 50399022, 'for': 50363295, 'admiral': 50357597, 'heis': 49469949, 'konkurs': 48269433, 'pantergaupe': 46717478, 'maldivisk': 49859434},
+            'Swedish': {'house': 50356446},
+            'Ancient Greek': {'ἀγγελία': 47719496}
         }
         for lang, words in words_to_test.items():
             parser.set_default_language(lang)
-            for word in words:
-                parsed_word = parser.fetch(word)
+            for word, oldid in words.items():
```

Comment:
Change oldid to oldId maybe?

Reply:
Why not `old_id` to follow [PEP 8] (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) ?

Reply:
That works too